src/ChainSafe.GamingSDK.EVM.Tests/ChainsafeRPCTests.cs

```diff
@@ -1,67 +1,93 @@
             */
+
+            // Deploy a contract with https://remix.ethereum.org/ and your Ganache instance.
+            // Make sure to select Ganache Provider as the Environment. Once deployed, replace the _contractAddress value with your contract address
+            // todo: This is impossible to automate
+            /* contractAddress = "0x2061c2B28F007DD0eD5064A61d352521CC1d2827"; */
         }
 
         [Test]
         public void GetNetworkTest()
         {
-            var network = infuraProvider.GetNetwork().Result;
-            Assert.AreEqual("Goerli", network.Name);
-            Assert.AreEqual(5, network.ChainId);
+            var network = web3.RpcProvider.GetNetwork().Result;
+            Assert.AreEqual("Geth Testnet", network.Name);
+            Assert.AreEqual(1337, network.ChainId);
         }
 
         [Test]
         public void GetBalanceTest()
         {
-            var balance = infuraProvider.GetBalance(goerliAddress).Result;
+            var balance = web3.RpcProvider.GetBalance(walletAddress).Result;
             var balanceFormatted = Units.FormatEther(balance);
             Assert.Greater(balanceFormatted, "0");
         }
 
         [Test]
         public void GetCodeTest()
         {
-            var code = infuraProvider.GetCode(goerliAddress).Result;
+            var code = web3.RpcProvider.GetCode(walletAddress).Result;
             Assert.AreEqual("0x", code);
         }
 
         [Test]
         public void GetLastBlockTest()
         {
-            var latestBlock = infuraProvider.GetBlock().Result;
+            var latestBlock = web3.RpcProvider.GetBlock().Result;
             Assert.AreEqual("0x0000000000000000", latestBlock.Nonce);
-            Assert.AreEqual("30000000", latestBlock.GasLimit.ToString());
+            Assert.AreEqual("6721975", latestBlock.GasLimit.ToString());
```

Comment:
how is this support to work?

Reply:
No idea. It always works on my machine, can't tell if it'd work with any Ganache instance.

Reply:
ah probably hardcoded in ganache instance

Reply:
I think those are the default values for a new ganache instance

Reply:
Default from the `anvil` "GoChain Testnet" now is used in this test,